com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/QueueHistoryFrame.cs

```diff
@@ -106,28 +107,39 @@ private FrameQueueItem GetCurrentQueueItem()
                 }
             }
 
+            m_CurrentQueueItem.updateStage = m_StreamUpdateStage;
+
             //Get the stream size
             m_CurrentQueueItem.streamSize = queueReader.ReadInt64();
 
-            //Inbound and Outbound message streams are handled differently
-            if (m_QueueFrameType == QueueFrameType.Inbound)
+            //Sanity checking for boundaries
+            if(m_CurrentQueueItem.streamSize < 131072 && m_CurrentQueueItem.streamSize > 0)
```

Comment:
I don't like magic numbers :( consider const variable?

Reply:
Added two mins and max stream bounds const values.